aptos-move/aptos-vm/src/lib.rs

```diff
@@ -144,6 +143,15 @@ pub trait VMValidator {
     ) -> VMValidatorResult;
 }
 
+/// This trait describes the VM's simulation interfaces.
+pub trait VMSimulator {
+    /// Executes a SignedTransaction without performing signature verification.
+    fn simulate_signed_transaction(
```

Comment:
Why don't we need a &self here? It seems that `VMSimulator` will be reused

Reply:
It is not reused right now, so I followed the same flow. It seems to me though we should, similarly for validator interface. Good point!